src/tapyrus-tx.cpp

```diff
@@ -32,6 +32,8 @@ static bool fCreateBlank;
 static std::map<std::string,UniValue> registers;
 static const int CONTINUE_EXECUTION=-1;
 
+const std::string CURRENCY_UNIT = "TPC";
```

Comment:
CURRENCY_UNIT is also defined in `feederate.cpp`. Similarly, in `tapyrusunits.cpp`. It would be good to define it in a common header file somewhere.

Reply:
CURRENCY_UNIT is defined in `feederate.cpp` and  'tapyrus-tx.cpp' for inclusion in 'tapyrusd' and 'tapyrus-tx'. As this is a string constant If I put it in a header, it would cause multiple definitions in each executable and linker error during compilation. I can add a new cpp file that can be included in both executables. but it is a lot more work just for one string.